client/src/views/ProfilePage.vue

```diff
@@ -204,9 +218,112 @@
 </template>
 
 <script>
-export default {};
+import {
+  Upload,
+  Form,
+  FormItem,
+  Input,
+  RadioGroup,
+  Radio,
+  Button,
+  Select,
+  Option
+} from 'element-ui';
+import { mapState, mapActions } from 'vuex';
+export default {
+  components: {
+    'el-upload': Upload,
+    'el-form': Form,
+    'el-form-item': FormItem,
+    'el-input': Input,
+    'el-radio-group': RadioGroup,
+    'el-radio': Radio,
+    'el-button': Button,
+    'el-select': Select,
+    'el-option': Option
+  },
+  data() {
+    return {
+      imageUrl: '',
+      ruleForm: {
+        username: '',
+        fullname: '',
+        info: {
+          PhoneNumber: '',
+          Email: '',
+          Address: '',
+          Sex: '',
+          Birthday: '',
+          Avatar: ''
+        }
+      },
+      rules: {
+        fullname: [
+          { required: true, message: 'Please input full name', trigger: 'blur' },
+          { min: 6, message: 'Length should be more 6', trigger: 'blur' }
```

Comment:
password must be at least 6 characters

Reply:
không anh ơi! em có 1 phần change password riêng còn đây chỉ là cái check thay đổi full name thôi ạ !